### PR TITLE
Updated Hololens2 conical grab pointers to default in MRTK 2.5

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Officially make the cone pointer standard for the hololens 2 profile in MRTK 2.5 and added documentation

## Changes
#7712
#7934

### CODE REVIEW

File: Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs [hunk 1]
```diff
@@ -479,6 +481,10 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
                         
                         if (grabbable != null)
                         {
+                            Debug.Log(grabbable);
```
Comment (julenka) [inline Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs:484]: Was this intentional?

File: Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs [hunk 2]
```diff
@@ -491,6 +497,11 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
             /// Returns true if any of the objects inside QueryBuffer contain a grabbable
             /// </summary>
             public bool ContainsGrabbable => grabbable != null;
+
+            /// <summary>
+            /// Returns true if any of the objects inside QueryBuffer contain a grabbable that is not a bounds handle
+            /// </summary>
+            public bool NearObjectDetected => ContainsGrabbable && !grabbable.IsBoundsHandles;
```
Comment (julenka) [inline Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs:504]: This looks like extra debug stuff also, maybe part of another pr?

File: Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs [hunk 3]
```diff
@@ -27,7 +27,7 @@ namespace Microsoft.MixedReality.Toolkit.Tests.Experimental
     /// <summary>
     /// Tests for runtime behavior of bounds control
     /// </summary>
-    public class BoundsControlTests
+    public class PlayModeBoundsControlTests
```
Comment (julenka) [inline Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs:30]: What's this for?

File: Documentation/Input/Pointers.md [hunk 4]
```diff
@@ -106,8 +106,9 @@ The *[SpherePointer](xref:Microsoft.MixedReality.Toolkit.Input.SpherePointer)* u
 Useful Sphere Pointer properties:
 
 - *Sphere Cast Radius*: The radius for the sphere used to query for grabbable objects.
-- *Near Object Margin*: The distance to query for detecting of an object is near the pointer.
+- *Near Object Margin*: The distance on top of the Sphere Cast Radius to query for detecting if an object is near the pointer. Total Near Object detection radius is Sphere Cast Radius + Near Object Margin
```
Comment (julenka) [inline Documentation/Input/Pointers.md:109]: Do we need to update any pictures here also to communicate that the default is a cone, not a sphere?

File: Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs (hunk 2 above)
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs:504]: Oh this is actually used in lines 171

public bool IsNearObject => queryBufferNearObjectRadius.ContainsGrabbable || queryBufferInteractionRadius.NearObjectDetected;

Its basically here to turn off near object interaction if the object is in the grabbable sphere cast range, since that edge case can occur when the near object detection is a sector. It also ignores bound handlers to avoid the bug we had experienced before with flickering handlers.

File: Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs (hunk 3 above)
Comment (RogPodge) [reply Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs:30]: Just a refactor because the clicking on the play mode boundscontroltests to open it would open up the edit mode boundscontroltests file instead.

File: Documentation/Input/Pointers.md (hunk 4 above)
Comment (RogPodge) [reply Documentation/Input/Pointers.md:109]: Will do
